### PR TITLE
initial commit to the UI of update order

### PR DESCRIPTION
**Pull request by haoruigao**
Add UI functionality for Update Order

### CODE REVIEW

File: features/orders.feature [hunk 1]
```diff
@@ -45,3 +45,22 @@ Scenario: List all Orders
...
+    And I should see "5" in the "Customer ID" field
+    And I should see "Confirmed" in the "status" dropdown
+    When I change "Customer ID" to "6"
+    And I select "Shipped" in the "status" dropdown
+    And I press the "Update" button
+    Then I should see the message "Success"
+    And I should see "6" in the "Customer ID" field
+    And I should see "Shipped" in the "status" dropdown
+    And I should see "Today's date" in the "Updated On" field
+    When I copy the "ID" field
```
Comment (MahamArif) [inline features/orders.feature:66]: Why you copy ID in the end?

Comment (haoruigao) [reply features/orders.feature]: wrong line, just delete, thanks for pointing out

File: features/orders.feature [hunk 2]
```diff
@@ -45,3 +45,21 @@ Scenario: List all Orders
...
+    Then I should see the message "Success"
+    And I should see "5" in the "Customer ID" field
+    And I should see "Confirmed" in the "status" dropdown
+    When I change "Customer ID" to "6"
+    And I select "Shipped" in the "status" dropdown
+    And I press the "Update" button
+    Then I should see the message "Success"
+    And I should see "6" in the "Customer ID" field
+    And I should see "Shipped" in the "status" dropdown
+    And I should see "Today's date" in the "Updated On" field
```
Comment (MahamArif) [inline features/orders.feature:65]: You should retrieve again to see updated values after clearing fields

**Review by MahamArif — APPROVED** [review]

File: features/orders.feature (hunk 2 above)
Comment (haoruigao) [reply features/orders.feature]: done, thanks!

**Pull request merged**